packages/text-area/src/TextArea.test.js

```diff
@@ -0,0 +1,60 @@
+import { mount } from "enzyme";
+import React from "react";
+import renderer from "react-test-renderer";
+import TextArea from "./TextArea";
+
+describe("TextArea", () => {
+  describe("integration", () => {
+    it("renders correctly", () => {
+      const tree = renderer.create(<TextArea label="HIG Text Area" />).toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+  });
+
+  describe("event handlers", () => {
+    let eventHandler;
+    const subject = (props = {}) =>
+      mount(<TextArea title="TextArea" {...props} />);
+    let wrapper;
+
+    beforeEach(() => {
+      eventHandler = jest.fn();
```

Comment:
_Nitpick_: This only needs to be called once. __We can either put it in a `before()` block, or just move it outside and use `const eventHandler = jest.fn();`